### ISSUE TITLE
should I try other resnet models?

### ISSUE THREAD

**Issue by dev2y** [fizyr/keras-retinanet#1517]
I was able to train a descent model with resnet50 and I am wondering if I should try resnet_100 and resnet150. Wanted to make sure before I commit to more compute cycles.

or should I try to improve resnet50 model with more images?

Thanks

**Comment by Hyper5phere** [comment]
Depends on your application. In general deeper models are more accurate but require more computation. In my use case I achieved ~9 % increase in mAP when I switched from resnet50 to resnet152, however, it is almost 2x times slower when inferencing. Also bear in mind that you might need to adjust your hyperparameters for deeper resnets, for example by lowering the learning rate slightly. But in the end, it's really difficult to say without testing with your specific data first. There's also other backbone architectures like VGG16 to try out, which might work better in your use case while not requiring additional processing time. Nevertheless, backbone (feature extractor) choice can have a significant impact on detector performance, so it's definitely worth experimenting with!
